frontend/packages/dev-console/integration-tests/features/operators.feature

```diff
@@ -126,3 +126,24 @@ Scenario: Install Web Terminal operator from Operator Hub page
    And user installs the Web Terminal operator with default values
    Then user will see a modal with title "Web Terminal"
    And user will see a View Operator button
+
+
+@regression, @smoke
+Scenario: Install Red Hat Integration - Camel Operator
+   Given user has installed OpenShift Serverless Operator
+   And user is at Operator Hub page with the header name "OperatorHub"
+   When user searches and installs the Red Hat Integration - Camel Operator with default values
+   Then user will see a modal with title "knative Apache Camel Operator"
+   And user will see a View Operator button
+
+
+@regression, @smoke
+Scenario: Create Integration Platform CR
+   Given user has installed OpenShift Serverless Operator
+   And user has installed Red Hat Integration - Camel Operator
+   And user is on Installed Operator page
+   When user clicks on Integration Platform link
+   And user clicks on Create IntegrationPlatform button
```

Comment:
this had relationship with namespace it's created in

Reply:
Added the namespace in the background which will run before every scenario.